content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+For monitoring purposes, it is absurd to use any timezone observing
```

Comment:
Add a link at the end: "For the author's real opinion on this topic, click
here".

On Thu, Feb 5, 2015 at 3:56 PM, Björn Rabenstein <notifications@github.com>
wrote:

> In content/docs/introduction/faq.md
> <https://github.com/prometheus/docs/pull/33#discussion_r24198602>:
>
> > @@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
> >  Yes, with [PromDash](/docs/visualization/promdash/) and [Console
> >  templates](/docs/visualization/consoles/).
> >
> > +### Can I change the timezone? Why is everything in UTC?
> > +
> > +For monitoring purposes, it is absurd to use any timezone observing
>
> I already wrote that a million times more benign than I feel about this
> scourge of humanity. I will dilute it down another trillion times...
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/docs/pull/33/files#r24198602>.
>